components/Activity/HeroCarousel.module.less

```
@@ -0,0 +1,167 @@
+.heroCarousel {
+  --hero-carousel-offset: 4.125rem;
+  background: #050816;
```

Stylelint (17.9.0)

[error] Expected empty line before declaration (declaration-empty-line-before)

(declaration-empty-line-before)